test/api/projects_api_test.rb

```diff
@@ -11,11 +11,33 @@ def app
   end
 
   def test_can_get_projects
-    user = FactoryBot.create(:user, :student, enrol_in: 0)
-    get with_auth_token('/api/projects', user)
+    # Create new student user
+    user = FactoryBot.create(:user, :admin)
+
+    # Create a dummy project
+    new_project = FactoryBot.create(:project)
+
+    # Perform Get
+    get with_auth_token "/api/projects/#{new_project.id}", new_project.user
+
+    # Check if the call success
```

Comment:
Add another check here please, check if the details from the GET match as expected